client/webserver/site/src/js/locales.ts

```diff
@@ -189,7 +191,9 @@ export const enUS: Locale = {
   [ID_MATCH_STATUS_REDEEM_PENDING]: 'Redeem PENDING',
   [ID_MATCH_STATUS_REDEMPTION_CONFIRMED]: 'Redemption Confirmed',
   [ID_MATCH_STATUS_COMPLETE]: 'Complete',
-  [ID_TAKER_FOUND_MAKER_REDEMPTION]: 'redeemed by {{ makerAddr }}'
+  [ID_TAKER_FOUND_MAKER_REDEMPTION]: 'redeemed by {{ makerAddr }}',
+  [ID_REFUND_COMING_YOUR_WAY]: 'Coming your way, just a little patience',
```

Comment:
Fine in English but a direct translation will likely be bonkers.  Maybe add a trailing comment like `// alt: Will happen in the next few blocks `

Reply:
Totally open to rephrase it however you like, added `alt` comment for now.